CONTRIBUTING.md

```diff
@@ -203,7 +208,7 @@ $ git push origin my-branch
 ```
 
 Go to https://github.com/yourusername/node and select your branch.
-Click the 'Pull Request' button and fill out the form.
+Click the 'Pull Request' button and [fill out the form](.github/PULL_REQUEST_TEMPLATE.md).
```

Comment:
This link is misleading. The text implies that you will be taken to a form. Instead, it will take you to a markdown file. I think this entire paragraph should be removed.